ogusa/tests/test_run_example.py

```diff
@@ -0,0 +1,43 @@
+#### import modules (same as run_ogusa_example)
+import multiprocessing
+from distributed import Client
```

Comment:
@jpycroft. I think you can get rid of the following import lines:
```python
...
from distributed import Client
...
import taxcalc
from taxcalc import Calculator
from ogusa import output_tables as ot
from ogusa import output_plots as op
from ogusa.execute import runner
from ogusa.constants import REFORM_DIR, BASELINE_DIR
from ogusa.utils import safe_read_pickle
```

Reply:
Thanks Rick. I've checked that it works without these lines, and now removed them. (An earlier solution I was playing needed them, but this one does not.)